src/components/Reports/TeamReport/components/UserLoginPrivileges.jsx

```diff
@@ -0,0 +1,135 @@
+import { ReportPage } from 'components/Reports/sharedComponents/ReportPage';
+import TeamReportLogs from './TeamReportLogs';
+import TeamsReportLogs from './TeamsReportLogs';
+import TeamReportCharts from './TeamReportCharts';
+import TeamsReportCharts from './TeamsReportCharts';
+import ReportCharts from './ReportCharts';
+
+function UserLoginPrivileges({ 
+  role,
+  teamName, 
+  teamMembers, 
+  totalTeamWeeklyWorkedHours,
+  selectedTeams, 
+  selectedTeamsWeeklyEffort,
+  allTeamsMembers }) {
+  // team
+  let teamWeeklyCommittedHours = 0;
+  let teamTotalBlueSquares = 0;
+
+  teamMembers.map(member => {
+    teamWeeklyCommittedHours += member.weeklycommittedHours;
+    teamTotalBlueSquares += member.infringements.length;
+  })
+
+  //selectedTeams
+  const [selectedTeamsMembers, setSelectedTeamsMembers] = useState([])
+
+  const [selectedTeamsData, setSelectedTeamsData] = useState([])
+  const [selectedTeamsTotalValues, setSelectedTeamsTotalValues] = useState({})
+
+  useEffect(() => {
+    const teamsData = selectedTeamsMembers.map((teamMembers, index) => {
+      const { totalCommitedHours, totalOfMembers, totalBlueSquares } = teamMembers.reduce((totals, member) => {
+        return {
+          totalCommitedHours: totals.totalCommitedHours + member.weeklycommittedHours,
+          totalOfMembers: totals.totalOfMembers + 1,
+          totalBlueSquares: totals.totalBlueSquares + member.infringements.length
+        };
+      }, { totalCommitedHours: 0, totalWorkedHours: 0, totalOfMembers: 0, totalBlueSquares: 0 });
+      
+      return {
+        name: selectedTeams[index]?.selectedTeam.teamName,
+        totalCommitedHours,
+        totalOfMembers,
+        totalBlueSquares
+      };
+    });
+  
+    setSelectedTeamsData(teamsData);
+  }, [selectedTeams, selectedTeamsMembers]);
+
+  useEffect(() => {
+    const { 
+      selectedTeamsTotalPeople, 
+      selectedTeamsTotalBlueSquares, 
+      selectedTeamsTotalCommitedHours, 
+    } = selectedTeamsData.reduce((totals, teamData) => {
+      return {
+        selectedTeamsTotalPeople: totals.selectedTeamsTotalPeople + teamData.totalOfMembers,
+        selectedTeamsTotalBlueSquares: totals.selectedTeamsTotalBlueSquares + teamData.totalBlueSquares,
+        selectedTeamsTotalCommitedHours: totals.selectedTeamsTotalCommitedHours + teamData.totalCommitedHours,
+      };
+    }, { selectedTeamsTotalPeople: 0, selectedTeamsTotalBlueSquares: 0, selectedTeamsTotalCommitedHours: 0 });
+    
+    setSelectedTeamsTotalValues({
+      selectedTeamsTotalPeople,
+      selectedTeamsTotalBlueSquares,
+      selectedTeamsTotalCommitedHours,
+    });
+  }, [selectedTeamsData]);
+
+  useEffect(() => {
+    const selectedTeamsMembersArray = selectedTeams.map((team) => allTeamsMembers[team.index]);
+    setSelectedTeamsMembers(selectedTeamsMembersArray);
+  }, [selectedTeams, allTeamsMembers]);
+  
+  // Check if the user has admin privileges
+  if (role == ('Administrator' || 'Owner')) {

```

Comment:
Please change the above check to` if (role === `Administrator` || role === `Owner` )`
Your current check only passes when the role is admin.... I used my owner account, got totally blank page there